tools/oesign/main.c

```diff
@@ -91,13 +86,27 @@ static const char _usage_sign[] =
     "  include:\n"
     "\n"
     "    Debug - whether enclave debug mode should be enabled (1) or not "
-    "(0)\n"
+    "(0). Defaults to 0\n"
     "    ProductID - the product identified number\n"
     "    SecurityVersion - the security version number\n"
     "    NumHeapPages - the number of heap pages for this enclave\n"
     "    NumStackPages - the number of stack pages for this enclave\n"
     "    NumTCS - the number of thread control structures for this "
     "enclave\n"
+    "    ExtendedProductID - a 128-bit globally unique identifier for the\n"
```

Comment:
Should note that this is SGX2 only (same for FamilyID, CapturePFGPExceptions, and KSS).